website/docs/how-to/how-to-troubleshoot-flag-exposure.md

```diff
@@ -0,0 +1,8 @@
+---
+title: My feature flag is enabled but all/some of our users are not exposed to it
+---
+
+To confirm how users will have flags resolved, follow these steps:
+1. The [Unleash Playground](../reference/playground.mdx Playground reference documentation) was developed with this particular use case in mind. An access token can be used along with context values (passed in via the UI) to see how a flag will be resolved. 
+2. When using a **gradual rollout** strategy, be mindful of the **[stickiness](../reference/stickiness)** value. When evaluating a flag, if the provided context does not include the field used in the stickiness configuration, the gradual rollout strategy will be evaluated to false.```
+3. If using [Unleash Edge](../reference/unleash-edge) or the [Proxy](../reference/unleash-proxy), consider changing the API endpoint being used by the SDK instance. The SDK and platform may be configured correctly while the intermediary (Edge or Proxy) is somehow malfunctioning. 
```

Comment:
@mvittiglio I was reviewing this and I'm not sure I understand:
1. Changing the API Endpoint to what value?
2. Saying that the intermediate is somehow malfunctioning doesn't help troubleshoot, and most likely it can be a misconfiguration more than a malfunction. 

So, maybe you're trying to say to use the [Frontend API](https://docs.getunleash.io/reference/front-end-api) from the SDK, to be able to tell if the problem is on the SDK or on the configuration of Edge/proxy, can that be?

For now, I'll remove this sentence, and we can add it back in another PR once clarified